### PR TITLE
WIP : new homepage with services

### PR DESCRIPTION
**Pull request by sseerrggii**
Only to know your opinion, @sauloperez @danypr92 @enricostano

Add services in homepage and some human photos :) Also update friends logos

NOTE: Need to reduce image size and translate keys to ES

Peek 27-06-2019 19-02

### CODE REVIEW

**Comment by enricostano** [comment]
Maybe there is no need to have a picture for each line. I would try something simpler.

Also, I think numbered lists are difficult to read when presenting this kind of things.

**Comment by sseerrggii** [comment]
I removed numbers. 

I'm not sure about images, I think with them we gain visibility as people, and if I should remove one I don't know witch one? Now is one per section. 

Peek 01-07-2019 13-25

**Comment by sauloperez** [comment]
I like it @sseerrggii ! Do you mind if I share a commit adding a bit of design on top? just to play a bit with margins, headers, etc.

File: _i18n/ca.yml [hunk 1]
```diff
@@ -8,10 +8,15 @@ menu:
...
+    description_2: "Si vols col·laborar amb nosaltres ens pots trobar al fòrum <a href='http://community.coopdevs.org'>community.coopdevs.org</a> i a les plataformes <a href='https://github.com/coopdevs'>github.com</a> i <a href='https://gitlab.com/coopdevs'>gitlab.com</a>."
+    description_3: "Si necessites els nostres serveis escriu-nos a <a href='mailto:info@coopdevs.org?subject=Hola%20Coopdevs%21&body=Estava%20mirant%20la%20vostra%20web%20i%E2%80%A6'>info@coopdevs.org</a> i passa a visitar-nos al <a href='https://passatgenogues.wixsite.com/cwnogues'>Coworking Nogués</a> de Gràcia."
+    action_lines: "Linies d'actuació"
+    open_source: "Dinamització de comunitats Open Source"
+    opensource_description: "Som “open by default” Desenvolupar software lliure amb una comunitat activa al voltant és la nostra passió, tenim experiència en la creació i dinamitació de comunitats Open Source. <a href='https://community.coopdevs.org/c/floss-communities'>Saber-ne més</a>"
```
Comment (sauloperez) [inline _i18n/ca.yml:15]: `Som “open by default”. Desenvolupem...`

File: _i18n/ca.yml [hunk 2]
```diff
@@ -8,10 +8,15 @@ menu:
...
+    development: "Desenvolupament de software per l'ESS"
```
Comment (sauloperez) [inline _i18n/ca.yml]: Desenvolupament de software per **a** l'ESS

File: _i18n/ca.yml [hunk 3]
```diff
@@ -8,10 +8,15 @@ menu:
...
+    development_description: "Desenvolupem software lliure per projectes de l'Economia Social i Solidària. Necessites ajuda? <a href='mailto:info@coopdevs.org?subject=Hola%20Coopdevs%21&body=Estava%20mirant%20la%20vostra%20web%20i%E2%80%A6'>Contacta amb nosaltres</a>"
```
Comment (sauloperez) [inline _i18n/ca.yml]: Desenvolupem software lliure per **a** projectes

File: _i18n/ca.yml [hunk 4]
```diff
@@ -8,10 +8,15 @@ menu:
...
+    processes: "Optimització de processos empresarials"
+    processes_description: "Analitzem i optimitzem processos empresarials per a cooperatives, fomentem la intercooperació en els processos que impliquen IT. <a href='https://community.coopdevs.org/c/it-processes-for-coops'>Saber-ne més</a>"
```
Comment (sauloperez) [inline _i18n/ca.yml]: what about replacing IT with "tecnologia" or "tecnologies de la informació"? Might easier to understand for non-techies.

Comment (raneq) [reply _i18n/ca.yml]: "_les TIC_" is also a common term

File: _i18n/ca.yml (hunk 1 above)
Comment (sseerrggii) [reply _i18n/ca.yml:15]: La forma verbal  ha de ser Desenvolupar per com segueix la frase

**Comment by sauloperez** [comment]
My proposal



So what do think @sseerrggii @raneq @danypr92 @enricostano @naoisegolden? I'll move to implementation if everyone agrees.

File: _i18n/ca.yml (hunk 1 above)
Comment (naoisegolden) [inline _i18n/ca.yml:15]:
```suggestion
    opensource_description: "Som <em>open by default</em>. Desenvolupar software lliure amb una comunitat activa al voltant és la nostra passió. Tenim experiència en la creació i dinamització de comunitats <em>Open Source</em>. <a href='https://community.coopdevs.org/c/floss-communities'>Saber-ne més</a>"
```

File: _i18n/ca.yml [hunk 5]
```diff
@@ -8,10 +8,15 @@ menu:
...
+    action_lines: "Linies d'actuació"
+    open_source: "Dinamització de comunitats Open Source"
+    opensource_description: "Som “open by default” Desenvolupar software lliure amb una comunitat activa al voltant és la nostra passió, tenim experiència en la creació i dinamitació de comunitats Open Source. <a href='https://community.coopdevs.org/c/floss-communities'>Saber-ne més</a>"
+    development: "Desenvolupament de software per a l'ESS"
+    development_description: "Desenvolupem software lliure per a projectes de l'Economia Social i Solidària. Necessites ajuda? <a href='mailto:info@coopdevs.org?subject=Hola%20Coopdevs%21&body=Estava%20mirant%20la%20vostra%20web%20i%E2%80%A6'>Contacta amb nosaltres</a>"
+    processes: "Optimització de processos empresarials"
+    processes_description: "Analitzem i optimitzem processos empresarials per a cooperatives, fomentem la intercooperació en els processos que impliquen les TIC. <a href='https://community.coopdevs.org/c/it-processes-for-coops'>Saber-ne més</a>"
```
Comment (naoisegolden) [inline _i18n/ca.yml:19]:
```suggestion
    processes_description: "Analitzem i optimitzem processos empresarials per a cooperatives. Fomentem la intercooperació en els processos que impliquen les TIC. <a href='https://community.coopdevs.org/c/it-processes-for-coops'>Saber-ne més</a>"
```

File: _i18n/es.yml [hunk 6]
```diff
@@ -7,11 +7,16 @@ menu:
...
+    description_2: "Si quieres colaborar con nosotros nos puedes encontrar en el foro <a href='http://community.coopdevs.org'>community.coopdevs.org</a> y en las plataformas <a href='https://github.com/coopdevs'>github.com</a> i <a href='https://gitlab.com/coopdevs'>gitlab.com</a>."
+    description_3: "Si necesitas nuestros servicios escríbenos a <a href='mailto:info@coopdevs.org?subject=Hola%20Coopdevs%21&body=Estava%20mirant%20la%20vostra%20web%20i%E2%80%A6'>info@coopdevs.org</a> y pasa a visitarnos en <a href='https://passatgenogues.wixsite.com/cwnogues'>Coworking Nogués</a> de Gràcia."
+    action_lines: "Líneas de actuación"
+    open_source: "Dinamización de comunidades Open Source"
+    opensource_description: "Somos “open by default“ Desarrollar software libre con una comunidad activa alrededor es nuestra pasión, tenemos experiencia en la creación y dinamitació de comunidades Open Source. <a href='https://community.coopdevs.org/c/floss-communities'>Saber más</a>"
```
Comment (naoisegolden) [inline _i18n/es.yml:15]:
```suggestion
    opensource_description: "Somos <em>open by default</em>. Desarrollar software libre con una comunidad activa alrededor es nuestra pasión. Tenemos experiencia en la creación y dinamización de comunidades <em>Open Source</em>. <a href='https://community.coopdevs.org/c/floss-communities'>Saber más</a>"
```

File: _i18n/es.yml [hunk 7]
```diff
@@ -7,11 +7,16 @@ menu:
...
+    development: "Desarrollo de software para la ESS"
+    development_description: "Desarrollamos software libre para proyectos de la Economía Social y Solidaria. Necesitas ayuda? <a href='mailto:info@coopdevs.org?subject=Hola%20Coopdevs%21&body=Estava%20mirant%20la%20vostra%20web%20i%E2%80%A6'>Contacta con nosotros</a>"
+    processes: "Optimización de procesos empresariales"
+    processes_description: "Analizamos y optimizamos procesos empresariales para cooperativas, fomentamos la intercooperación en los procesos que implican las TIC. <a href='https://community.coopdevs.org/c/it-processes-for-coops'>Saber más/a>"
```
Comment (naoisegolden) [inline _i18n/es.yml:19]:
```suggestion
    processes_description: "Analizamos y optimizamos procesos empresariales para cooperativas. Fomentamos la intercooperación en los procesos que implican las TIC. <a href='https://community.coopdevs.org/c/it-processes-for-coops'>Saber más/a>"
```

**Review by naoisegolden — CHANGES_REQUESTED** [review]
Looks super good @sauloperez !

I added some formatting suggestions to the copy for your consideration. Also some typos fixes.

Also, the Coopdevs logo in the footer should have a transparent background.

**Comment by sauloperez** [comment]
All the proposed changes have been taken into account in the new design. We were only missing https://github.com/coopdevs/coopdevs.github.io/pull/75.

**Pull request closed without merging**